velox/functions/prestosql/DateTimeFunctions.h

```diff
@@ -1499,4 +1504,39 @@ struct ToISO8601Function {
   }
 };
 
+template <typename T>
+struct TimestampWithTimezoneAtTimezoneFunction
+    : public TimestampWithTimezoneSupport<T> {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  std::optional<int64_t> targetTimezoneID_;
+
+  FOLLY_ALWAYS_INLINE void initialize(
+      const core::QueryConfig& config,
+      const arg_type<TimestampWithTimezone>* /*tsWithTz*/,
+      const arg_type<Varchar>* timezone) {
+    if (timezone != nullptr) {
+      targetTimezoneID_ = util::getTimeZoneID(
+          std::string_view(timezone->data(), timezone->size()));
```

Comment:
nit: I think implicit conversion from StirngView into std::string_view is allowed, IIRC. Can't you just use `*timezone` here? (or `(std::string_view)*timezone`)

Reply:
Thanks, explicit cast did work. Replaced.